### PR TITLE
Add atom & phpstom plugins

### LINKED ISSUE #187: add atom plugin information

### PR DESCRIPTION
**Pull request by Grummfy**
fix #187 #186

I have also added some link reference.

Thanks for reviewing it.

### CODE REVIEW

File: source/en/ide.rst [hunk 1]
```diff
@@ -90,19 +94,71 @@ For more details on the use of plug-in, help is available in VIM with the follow
...
+
+It's easy to install, simply follow the `official documentation <http://flight-manual.atom.io/using-atom/sections/atom-packages/>`_ or theses steps:
+
+* Open atom
+* Go to *Settings*, then click on *Install*
+* Search for *atoum* in the list, then click on the install button
+
+If you need more information check the `repository of the package <https://github.com/atoum/atom-plugin>`_.
+
+
```
Comment (Hywan) [inline source/en/ide.rst]: Too much empty lines.

File: source/en/ide.rst [hunk 2]
```diff
@@ -90,19 +94,71 @@ For more details on the use of plug-in, help is available in VIM with the follow
...
+* Open PhpStorm
+* Go to *File -> Settings*, then click on *Plugins*
+* Click on Browse repositories
+* Search for *atoum* in the list, then click on the install button
+* Restart PhpStorm
+
+If you need more information check the `repository of the plugins <https://github.com/atoum/phpstorm-plugin>`_.
+
+
```
Comment (Hywan) [inline source/en/ide.rst]: Too much empty lines.

File: source/en/ide.rst (hunk 1 above)
Comment (Grummfy) [inline source/en/ide.rst]: I'm not sure it change anything. But yeah, I will remove it.

**Comment by Grummfy** [comment]
fixed

**Comment by mikaelrandy** [comment]
👍

**Pull request merged**